app/src/main/java/de/danoeh/antennapod/fragment/SubscriptionFragment.java

```diff
@@ -66,6 +69,8 @@
     private static final String PREFS = "SubscriptionFragment";
     private static final String PREF_NUM_COLUMNS = "columns";
 
+    public static final int SWIPE_TO_REFRESH_DURATION = 750;
```

Comment:
Is there any class where constants like this one are saved, I did not find an appropriate one.
Furthermore I changed the duration to 750ms which is the time for one repetition.

Reply:
Good question. I don't think we have a class for such constants. The most Android way would probably be to use the `core/res/values/integers.xml` file and then load with `getResources().getInteger(R.integer.xy)`